src/components/generics/vacation-test-forms/my-vacation-requests.tsx

```diff
@@ -50,8 +54,10 @@ const RenderVacationRequests = () => {
 
     try {
       const vacationsApi = Api.getVacationRequestsApi(accessToken?.access_token);
+      // Use your own keyCloakId, if you wish to display vacation requests created by you
+      // It is currently a limitation of the TimeBankApi. It puts your keyCloakId in the
+      // vacation request when creating a new vacation request, regardless of the Id you use in createVacationRequest()
```

Comment:
If I understand this correctly, does this mean we can only create vacationRequests for the currently logged in user, regardless of the id that is passed in to the createVacationRequest? Perhaps this isn't a limitation, although I can see why it would be good to override this for development purposes. Lets have a chat about this. Perhaps this comment can be removed

Reply:
Yes, we can only create vacationRequests for the currently logged in user, maybe it's supposed to be that way for production but troublesome for development.